SharpSnmpLib/ByteTool.cs

```diff
@@ -127,26 +148,75 @@ public static string Convert(byte[] buffer)
             return BitConverter.ToString(buffer).Replace('-', ' ');
         }
 
-        internal static byte[] ParseItems(params ISnmpData[] items)
+        internal static byte[] ParseItems(ISnmpData item1, ISnmpData item2, ISnmpData item3, ISnmpData item4)
         {
-            if (items == null)
+            var buffer = ArrayPool<byte>.Shared.Rent(1024 * 1024);
+
+            using (var result = new MemoryStream(buffer, 0, buffer.Length))
```

Comment:
Very unlikely it will throw given 1MB buffer allocation but just in case maybe we should catch `System.NotSupportedException: 'Memory stream is not expandable.'` and behave the old way

Reply:
https://github.com/ravendb/ravendb/blob/91a7a00361a6599ece68d90f3044a0231c642d8d/src/Raven.Server/Monitoring/Snmp/SnmpWatcher.cs#L214-L218

I would prefer to see it there.

Reply:
I'm fine with that